starboard/build/config/BUILDCONFIG.gn

```diff
@@ -451,6 +451,88 @@ template("shared_library") {
       }
     }
 
+    # Add an additional evergreen_loader target to every gtest target.
+    if (sb_is_evergreen_compatible && sb_evergreen_compatible_package &&
+        target_name != "loader_app") {
+      original_target_name = target_name
+      data_deps += [ "//third_party/icu:icudata" ]
+      data_deps += [
+        ":copy_loader_app_lib${original_target_name}",
+        "//starboard/loader_app:copy_loader_app_content",
+      ]
+      if (cobalt_font_package == "empty") {
+        data_deps += [ "//cobalt/content/fonts:copy_font_data" ]
+      } else {
+        data_deps += [
+          "//cobalt/content/fonts:copy_fonts",
+          "//cobalt/content/fonts:fonts_xml",
+        ]
+      }
+      shared_library(original_target_name + "_evergreen_loader") {
```

Comment:
I'd prefer to define this target in another file if possible (possibly with a template). Even better, that template can define all 4 targets

Reply:
Could you explain the cyclic dependency to me?